spec/kit_spec.rb

```diff
@@ -0,0 +1,38 @@
+require_relative '../lib/all'
+require 'stringio'
+require 'httpclient'
+
+class HTTPClientDownloadNeededPart < StandardError; end
+
+describe 'kit_spec' do
+  Dir['./lib/engines/hckinstall/kits/*.json'].each do |json_file|
+    kit_url = AutoHCK::Models::Kit.from_json_file(json_file).download_url
+    if kit_url.nil?
+      print("No download URL for #{json_file}\n")
```

Comment:
I'm worried that this may get printed for all test runs even when you run irrelevant tests.

Reply:
why?
is this better to use it + pending in this case?